### ISSUE TITLE
KDE freezes when copying to clipboard - still happening

### ISSUE THREAD

**Issue by noraj** [flameshot-org/flameshot#551]
**Flameshot version**

Flameshot v0.6.0
Compiled with Qt 5.12.1

Installed via [AUR](https://aur.archlinux.org/packages/flameshot-git/)

**Describe the bug**

After a screenshot is taken, I click on the copy to clipboard button (flameshot gui) and the whole UI freeze.

I'm forced to change TTY and kill the process.

This is what appears during the manipulation:

```
$ flameshot       
QPainter::begin: Paint device returned engine == 0, type: 2
QPainter::setRenderHint: Painter must be active to set rendering hints
QPainter::setCompositionMode: Painter not active
QPainter::translate: Painter not active
QPainter::setPen: Painter not active
QPainter::setBrush: Painter not active
QPainter::setBrush: Painter not active
[1]    13645 killed     flameshot
```

I don't think  #161 is related, but #389 may be and #170 seems to be exactly the same problem as me.

The weirdest things is that it is not systematic.

I can copy to clipboards tons of time nothing happens and then another copy to clipboard and it freeze, and when it start freezing and can kill and start it back it will freeze each time.

**To Reproduce**

I don't know what to say more than the description.

**Expected behavior**

No crash obviously.

**System Information**

Manjaro with KDE.

```
$ uname -a
Linux manji 4.19.45-1-MANJARO #1 SMP PREEMPT Wed May 22 17:16:41 UTC 2019 x86_64 GNU/Linux

$ plasmashell --version
plasmashell 5.15.5
```

---

Update: this is very annoying to have to switch to a TTY to kill the flameshot process.

**Comment by rindek** [comment]
Happens same in Manjaro for some time on XFCE

```
Flameshot v0.6.0
Compiled with Qt 5.12.1

Linux Carbon 5.2.11-1-MANJARO #1 SMP PREEMPT Thu Aug 29 07:41:24 UTC 2019 x86_64 GNU/Linux
```

Flameshot freezes whole UI after doing ANY action (even cancelling capture). The only way to unstuck this without killing flameshot process is to switch to TTY and back again to xorg

**Comment by noraj** [comment]
Works flawlessly under ArchLinux, but Freeze very often under Manjaro.

**Comment by CiderAndWhisky** [comment]
The same happens to me (Ubuntu 2020.04, pretty new installation). Anyone found a solution for this?

**Comment by avindra** [comment]
Check out issue #635 which is probably related. I'm not using KDE at all, which may be my problem in openSUSE Tumbleweed.

**Comment by CiderAndWhisky** [comment]
https://github.com/lupoDharkael/flameshot/issues/635#issuecomment-625276332 solved my problem. Thanks!

**Comment by borgmanJeremy** [comment]
If #635 resolves the issue then the newer 0.8.x release will resolve this issue. It was a race condition related to notifications and DBUS. Please report back if it happens again. Thanks!
